lib/manager/jsonnet-bundler/artifacts.ts

```diff
@@ -0,0 +1,102 @@
+import { quote } from 'shlex';
+import { TEMPORARY_ERROR } from '../../constants/error-messages';
+import { logger } from '../../logger';
+import { ExecOptions, exec } from '../../util/exec';
+import { readLocalFile } from '../../util/fs';
+import { getRepoStatus } from '../../util/git';
+import { regEx } from '../../util/regex';
+import type {
+  PackageDependency,
+  UpdateArtifact,
+  UpdateArtifactsResult,
+} from '../types';
+
+function dependencyUrl(dep: PackageDependency): string {
+  const url = dep.lookupName;
+  if (dep.managerData?.subdir) {
+    return url.concat('/', dep.managerData.subdir);
+  }
+  return url;
+}
+
+export async function updateArtifacts(
+  updateArtifact: UpdateArtifact
+): Promise<UpdateArtifactsResult[] | null> {
+  const { packageFileName, updatedDeps, newPackageFileContent, config } =
+    updateArtifact;
+  logger.trace(`jsonnetfile.updateArtifacts(${packageFileName})`);
+  logger.trace({ newPackageFileContent });
+
+  const lockFileName = packageFileName.replace(regEx(/\.json$/), '.lock.json');
+  const existingLockFileContent = await readLocalFile(lockFileName, 'utf8');
+
+  if (!existingLockFileContent) {
+    logger.debug('No jsonnetfile.lock.json found');
+    return null;
+  }
+
+  try {
+    const execOptions: ExecOptions = {
+      cwdFile: packageFileName,
```

Comment:
```suggestion
      cwdFile: packageFileName,
      docker: {
         image: 'sidecar',
      },
```
Please add a `ToolConstraint` for jb like composer. If we can't extract the version somehow, simply pass `config.constraints?.jb`.

https://github.com/renovatebot/renovate/blob/4e07ddf52652445024a78a04dd70bc34d6be4d19/lib/manager/composer/artifacts.ts#L99-L121

https://github.com/renovatebot/renovate/blob/04620d71a8650e742e2eb3bbbf0e778b63554f1d/lib/util/exec/buildpack.ts#L8-L14